lib/instances/GuiService.lua

```diff
@@ -1,4 +1,33 @@
 local BaseInstance = import("./BaseInstance")
 local GuiService = BaseInstance:extend("GuiService")
 
+local function validateType(name, value, expectedTypeAsString)
+	local actualType = type(value)
+	if actualType ~= expectedTypeAsString then
+		error(string.format("%s must be type `%s`, got type `%s`", name, expectedTypeAsString, actualType), 3)
+	end
+end
```

Comment:
I understand the reason for this function, but having to do this still makes me depressed.